packages/app/src/app/overmind/namespaces/workspace/actions.ts

```diff
@@ -116,6 +116,31 @@ export const tagsChanged: AsyncAction<{
   await actions.workspace.tagAdded();
 };
 
+/** tagsChanged2 takes new tags and does the diffing on its own
+ * This is v2 of tagsChanged. It's used in the redesign
+ */
+export const tagsChanged2: AsyncAction<string[]> = withOwnedSandbox(
```

Comment:
Duuuude, the naming, no 😂

maybe call it tagsUpdate or something, or add this to changed and only run it if the new sidebar is in place

Reply:
I was hoping to call it _new_ and then delete the old one once we migrate everyone?